src/Command/ImportQuestionnairesCommand.php

```diff
@@ -0,0 +1,291 @@
+    }
+
+    public function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $output->writeln("Starting import of questionnaires...");
+
+        if (!file_exists($this->pathToJson)) {
+            $output->writeln("No data to import. File at " . $this->pathToJson . " not found.");
+            return 1;
+        }
+
+        $questionnaires = JsonMachine::fromFile($this->pathToJson);
+        foreach ($questionnaires as $questionnaire) {
+            $this->importQuestionnaire($questionnaire);
+        }
+
+        $output->writeln("Questionnaire import process has finished.");
+        return 0;
+    }
+
+    /**
+     * @param $questionnaire
+     */
+    private function importQuestionnaire($questionnaire): void
+    {
+        $db_questionnaire = $this->questionnaireRepo->findOneBy(["type" => $questionnaire["type"]]);
+
+        if (!$db_questionnaire) {
+            $db_questionnaire = new Questionnaire();
+            $db_questionnaire->setType($questionnaire["type"]);
+        }
+
+        $aspects = $questionnaire["aspects"];
+
+        $this->em->persist($db_questionnaire);
+        $this->em->flush();
+
+        foreach ($aspects as $aspect) {
+            $isDeprecated = false;
+            if (array_key_exists("deprecated", $aspect)) {
+                $isDeprecated = $aspect["deprecated"];
+            }
+
+            $this->importAspect($aspect, $db_questionnaire, $isDeprecated);
+        }
+    }
+
+    private function importAspect($aspect, Questionnaire $questionnaire, $isDeprecated = false)
+    {
+        $db_aspect = $this->aspectRepo->findOneBy([
+            "questionnaire" => $questionnaire->getId(),
+            "local_id" => $aspect["id"]
+        ]);
+
+        if (!$db_aspect) {
+            $db_aspect = new Aspect();
+            $db_aspect->setCreatedAt(new \DateTimeImmutable("now"));
+            $db_aspect->setLocalId($aspect["id"]);
+        }
+
+        if ($isDeprecated) {
+            $db_aspect->setDeletedAt(new \DateTimeImmutable("now"));
+            $this->em->persist($db_aspect);
+            return;
+        }
+
+        $db_aspect->setNameDe($aspect["name_de"]);
+        $db_aspect->setNameFr($aspect["name_fr"]);
+        $db_aspect->setNameIt($aspect["name_it"]);
+        $db_aspect->setDescriptionDe(array_key_exists("description_de", $aspect) ? $aspect["description_de"] : "");
+        $db_aspect->setDescriptionFr(array_key_exists("description_fr", $aspect) ? $aspect["description_fr"] : "");
+        $db_aspect->setDescriptionIt(array_key_exists("description_it", $aspect) ? $aspect["description_it"] : "");
+        $db_aspect->setQuestionnaire($questionnaire);
+
+        $this->em->persist($db_aspect);
+        $this->em->flush();
+
+        $questions = $aspect["questions"];
+
+        foreach ($questions as $question) {
+            $questionIsDeprecated = $isDeprecated;
+            if (array_key_exists("deprecated", $question)) {
```

Comment:
Same thing as above the value could end up being NULL since you are only checking if the key exists, if the entity field is not `nullable=true` then you will have a problem when saving it to DB.